frontend/src/components/ControlPanel/ControlPanel.css

```diff
@@ -35,3 +35,8 @@
 	border-bottom-right-radius: 0;
 	border-bottom-left-radius: 0;
 }
+
+.error-message {
```

Comment:
"error-message" is a pretty generic name for a css class, so this style declaration could "leak" into other components that happen to use the same classname. Better to make the selector more specific:

```
.control-panel .control-collapsible-section .error-message {
```